test/app-tests/loading.js

```diff
@@ -343,6 +346,40 @@ describe('loading:', function () {
             }).done(done, done);
         });
 
+        it("registers correctly with a Riot Team Server", function() {
+            sdk.setFetch(httpBackend.fetchFn); // XXX: ought to restore this!
```

Comment:
Also, I haven't convinced myself how this is getting used but I shall look when I get in this morning!